### DISCUSSION TITLE
Kedro and Docker

### DISCUSSION THREAD

**Discussion by nchelaru** [kedro-org/kedro#1007] [Idea]
Hello!

First of all, fantastic work on Kedro, I can't imagine my team's work without it!

I'm currently exploring docker as an option for sharing Kedro pipelines between team members and deployment to our production server. I noticed that the `kedro-docker` plugin has not been updated since January 2021 and recent issues have not been addressed by the developers. 

Does anyone use Kedro+Docker frequently in their own workflows? I would love to hear more about people's experiences on this front. As far as I can see, I have not found anyone write about this in detail.

Thank you!

**Comment by datajoely** [comment]
Hi @nchelaru  as discussed on [Discord](https://discord.com/channels/778216384475693066/846330075535769601/903799583199592458) docker is very common in Kedro workflows. The kedro-docker plugin aims to get people up and running quickly, but is by no means required for using Kedro in a container.

The plugin is still being maintained, we will look into the issue reported. Ultimately, however, Kedro just needs a python environment and access to the configuration / datasets defined within it to work.

**Reply by nchelaru** [reply]
Fantastic, thank you @datajoely for the reply here and on Discord! :)

I am a total newbie at Docker and fairly experienced with Kedro, and had no problem setting up a Docker container for my Kedro pipeline using a Dockerfile. So, really easy for anyone else thinking of doing the same thing!
